7.2/alpine/Dockerfile

```diff
@@ -1,20 +1,28 @@
 FROM php:7.2-alpine
-LABEL maintainer MilesChou <jangconan@gmail.com>
 
-ENV PHALCON_VERSION=3.3.1
-ENV PHALCON_DEV_TOOLS_VERSION=3.2.12
+LABEL maintainer="MilesChou <github.com/MilesChou>"
+LABEL maintainer="fizzka <github.com/fizzka>"
+
+ARG PHALCON_VERSION=3.3.1
+ARG PHALCON_DEV_TOOLS_VERSION=3.2.12
+ARG PHALCON_EXT_PATH=php7/safe
```

Comment:
1) are you sure about "safe"?
2) i think i needs to do additional calculation for php5

Reply:
I not sure. I don't know the difference between "safe" and the others. Should I use 64bits?

Reply:
OK, I will follow Phalcon native install. :)